legal-api/src/legal_api/services/permissions.py

```diff
@@ -0,0 +1,49 @@
+# Copyright © 2025 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=too-many-lines
+"""This manages all of the authentication and authorization service."""
+from http import HTTPStatus
+
+from flask import jsonify
+
+from legal_api.models.authorized_role_permission import AuthorizedRolePermission
+
+class PermissionService:
+    """Service to manage permissions for user roles."""
+
+    @staticmethod
+    def get_authorized_permissions_for_user():
+        """
+        Returns a JSON response containing the authorized permissions for the current user.
+        """
+        from legal_api.services.authz import cache, get_authorized_user_role  # pylint: disable=import-outside-toplevel
```

Comment:
can you move `get_authorized_user_role` function to `permissions.py` and `cache = Cache()` to https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/__init__.py to avoid cyclic reference